<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_value_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_value_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_number_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Число 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_number_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Число 2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sum">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Сложение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result_text">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_result_value">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Нет</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reduce">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычитание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Умножение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Деление</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_math_in">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>40</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Считать в:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_grad">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Градусы</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_rad">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Радианы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ctg">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>ctg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tg">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>tg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arcsin">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>arcsin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arccos">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>arccos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
